### DISCUSSION TITLE
How to change Nodeport

### DISCUSSION THREAD

**Discussion by josetemitayoHH96** [k3s-io/k3s#8050] [Q&A] [answered]
I am trying to change nodeport on k3s and I tried this command
**k3s server --kube-apiserver-arg --service-node-port-range=20618-20828**
but I got this error

FATA[0000] starting kubernetes: preparing server: init cluster datastore and https: listen tp :6443: bind: address already use

**Comment by brandond** [comment, marked as answer]
You appear to be trying to run another copy of k3s, instead of modifying the CLI flags for the service. Use `systemctl edit --full k3s` to edit the systemd unit, make your changes to the CLI flags, and then restart the service.

Also, the format you want for that flag is`--kube-apiserver-arg=service-node-port-range=20618-20828`.
